extensions/neodb/src/components/BookMeta.tsx

```diff
@@ -18,7 +18,7 @@ const BookMeta: React.FC<BookMetaProps> = ({ data, rating }) => {
       {translator.length !== 0 && <Detail.Metadata.Label title="Translator" text={translator.join(", ")} />}
       {language && <Detail.Metadata.Label title="Subtitle" text={language} />}
       {pub_house && <Detail.Metadata.Label title="Publication" text={pub_house} />}
-      {pub_year && <Detail.Metadata.Label title="Publication" text={pub_year.toString()} />}
+      {pub_year && <Detail.Metadata.Label title="Published Time" text={pub_year.toString()} />}
```

Comment:
maybe just `Published` would be better? `Time` is usually used for more detailed `date`